### ISSUE TITLE
Add redirect to first unit in a course when unit slug not included

### ISSUE THREAD

**Issue by yoshiemuranaka** [Qiskit/saiba#345]
### Feature description

Currently you need to enter in the full unit path in order to see a specific course. eg `/course/<course-slug>/<unit-slug>`. Going to `/course/<course-slug>` renders a 404.

It looks like in platypus we have some kind of redirect in place so that navigating to `/course/<course-slug>` would redirect to the first unit `/course/<course-slug>/<first-unit-slug>`

It would be nice to add something similar to saiba

**Comment by abdonrd** [comment]
At some point, I think the idea is to show a course landing page.
Something like: https://qiskit.org/learn/course/basics-quantum-information

**Comment by vabarbosa** [comment]
yes, the idea is to move the course landing page from qiskit.org into learn.qiskit.org ... however, i do not see that happening soon so until it does it would be nice to redirect to the first unit in the course
